src/USER-HSMA/HSMA_Example/salt_1-1.in

```diff
@@ -0,0 +1,79 @@
+package omp 40
+
+processors * * * grid numa
+log FMM.out

```

Comment:
Please do not use the "log" command in example inputs

Reply:
Resolved